src/main/java/edu/harvard/iq/dataverse/DataversePage.java

```diff
@@ -512,15 +512,18 @@ public void updateInclude(Long mdbId, long dsftId) {
         List<SelectItem> retList = new ArrayList<>();
         if ((typeIn.isHasParent() && typeIn.getParentDatasetFieldType().isInclude()) || (!typeIn.isHasParent() && typeIn.isInclude())) {
             SelectItem requiredItem = new SelectItem();
+            //requiredItem.setLabel(BundleUtil.getStringFromBundle("dataverse.item.required"));
             requiredItem.setLabel("Required");
             requiredItem.setValue(true);
             retList.add(requiredItem);
             SelectItem optional = new SelectItem();
+            //optional.setLabel(BundleUtil.getStringFromBundle("dataverse.item.optional"));
             optional.setLabel("Optional");
             optional.setValue(false);
             retList.add(optional);
         } else {
             SelectItem hidden = new SelectItem();
+            //hidden.setLabel(BundleUtil.getStringFromBundle("dataverse.item.hidden"));
             hidden.setLabel("Hidden");
```

Comment:
What's the plan for "Required" and "Hidden" above? They're still hard coded as English. Do we plan to move to the version in Bundle.properties such as "dataverse.item.hidden".?

Reply:
I would like to move it to bundle.properties, but wondering whether it will affect any other application logic.